packages/documentation/.storybook/addons/styles-switcher/StylesSwitcher.tsx

```diff
@@ -73,7 +73,7 @@ function StylesSwitcher() {
 
     observer = new MutationObserver(
       debounce(() => {
-        setStories(preview.querySelectorAll('.sbdocs-preview, .sb-main-padded'));
+        setStories(preview.querySelectorAll('.sbdocs-preview'));
```

Comment:
```suggestion
        setStories(preview.querySelectorAll('.sbdocs-preview, .sb-main-padded'));
```

This is needed to change the scheme on the full page preview.